source/ptreferential/ptref_graph.cpp

```diff
@@ -145,6 +145,7 @@ Jointures::Jointures() {
     boost::add_edge(vertex_map.at(Type_e::Dataset), vertex_map.at(Type_e::Route), Edge(2), g);
     // From a StopPoint we can have datasets
     boost::add_edge(vertex_map.at(Type_e::Dataset), vertex_map.at(Type_e::StopPoint), Edge(3), g);
+    boost::add_edge(vertex_map.at(Type_e::Dataset), vertex_map.at(Type_e::Network), Edge(3), g);
```

Comment:
gnagnagna

On Thu, Sep 8, 2016 at 8:39 AM, Alexandre Jacquin <notifications@github.com>
wrote:

> In source/ptreferential/ptref_graph.cpp
> <https://github.com/CanalTP/navitia/pull/1754#discussion_r77952801>:
>
> > @@ -145,6 +145,7 @@ Jointures::Jointures() {
> >      boost::add_edge(vertex_map.at(Type_e::Dataset), vertex_map.at(Type_e::Route), Edge(2), g);
> >      // From a StopPoint we can have datasets
> >      boost::add_edge(vertex_map.at(Type_e::Dataset), vertex_map.at(Type_e::StopPoint), Edge(3), g);
> > +    boost::add_edge(vertex_map.at(Type_e::Dataset), vertex_map.at(Type_e::Network), Edge(3), g);
>
> I don't care about the saying of a bot!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CanalTP/navitia/pull/1754/files/cadd5f9ae6e5bcc5e916f0a5ef2f8cdc6cad8166#r77952801>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADzY8n5CQL_9MCmk8FoaM-AI73f6_2KOks5qn620gaJpZM4J3D8t>
> .
>